third_party/ruby/ruby.BUILD

```diff
@@ -33,7 +36,11 @@ ruby(
     cppopts = [
         "-Wdate-time",
         "-D_FORTIFY_SOURCE=2",
-    ],
+    ] + select({
+        # Don't include JIT statistics unless we're building JIT support
+        "@com_stripe_ruby_typer//tools/config:jit_enabled": ["-DYJIT_STATS=1"],
```

Comment:
This is not turned on by default?  Is there a performance penalty associated with collecting the statistics, and therefore we might not want to have statistics turned on just because we have the jit turned on?

Reply:
Works for me!